steam_wrapper/steam_wrapper.py

```diff
@@ -154,7 +154,8 @@ def migrate_data():
     xdg_data_home = os.path.join(STEAM_ROOT, target)
     if not os.path.islink(source):
         if os.path.isdir(target):
-            copytree(target, backup)
+            copytree(target, backup,
+                     ignore=[os.path.join(target, "Steam")])
```

Comment:
`target` here is relative path. Will `fnmatch()` handle it correctly?

Reply:
It should. fnmatch won't care even if the file doesn't exist.

Reply:
Looks fine to me, then.